test/unittests/execute_call_test.cpp

```diff
@@ -361,14 +363,14 @@ TEST(execute_call, imported_function_call_void)
 
     const auto module = parse(wasm);
 
-    bool called = false;
-    const auto host_foo = [&called](Instance&, const Value*, int) {
+    static bool called = false;
```

Comment:
Hmm, why not just capture this in the function below?

Reply:
Lambda's capture can't be used, because only lambdas without capture can be converted into function pointer.

Another option is to pass it via `host_context`, but I decided against it, see https://github.com/wasmx/fizzy/pull/716/files/5baa695073c325728d1e60ee321ef94bf77e7248#r580939050

Also a possible evolution of this that would allow lambdas with capture is in #740